### PR TITLE
Fixed the columnProcess Lineage and a bug

### PR DESCRIPTION
**Pull request by hr2904**
## Change description

> Fixed the columnProcess Lineage and  a bug which was causing 2 simultaneous sync and async tasks being executed for the same tag propagation.

## Type of change
- [ ] Bug fix (fixes an issue)

## Related issues

> Fix [Tag Propagation through lineage and hierarchy - ColumnProcess](https://atlanhq.atlassian.net/browse/PLT-1092) 

### Development

- [ ] Lint rules pass locally
- [ ] Application changes have been tested thoroughly
- [ ] Automated tests covering modified code pass

### Security

- [ ] Security impact of change has been considered
- [ ] Code follows company security practices and guidelines

### Code review 

- [ ] Pull request has a descriptive title and context useful to a reviewer. Screenshots or screencasts are attached as necessary
- [ ] "Ready for review" label attached and reviewers assigned
- [ ] Changes have been reviewed by at least one other contributor
- [ ] Pull request linked to task tracker where applicable

### CODE REVIEW

**Review by checkaayush — APPROVED** [review]
LGTM. Please get an approval from Suman/Anshul before merging.

**Review by sumandas0 — APPROVED** [review]
Looks fine

**Pull request merged**
